rotation.go

```diff
@@ -229,7 +229,7 @@ func (b *backend) rotateCredential(ctx context.Context, s logical.Storage) bool
 
 	resp, err := b.setStaticAccountPassword(ctx, s, input)
 	if err != nil {
-		b.Logger().Error("unable to rotate credentials in periodic function", "error", err)
+		b.Logger().Error("unable to rotate credentials in periodic function", "rotationID", item.Key, "error", err)
```

Comment:
Should we be consistent and either use `err` or `error`?

Reply:
moved to "error" per discussion